ouroboros-network/src/Cardano/Node/PeerSelection/Governor/PeerSelectionState.hs

```diff
@@ -0,0 +1,55 @@
+module Cardano.Node.PeerSelection.Governor.PeerSelectionState where
+
+import Cardano.Node.ConsensusMode (ConsensusMode)
+import Cardano.Node.PeerSelection.Bootstrap (UseBootstrapPeers (..))
+import Cardano.Node.Types (LedgerStateJudgement (..),
+           MinBigLedgerPeersForTrustedState (..))
+import Control.Monad.Class.MonadTime.SI (Time)
+
+-- | Cardano Node PeerSelection State extension data type.
+-- It contain specific PeerSelection state parameters to guide the Outbound
+-- Governor.
+--
+data CardanoPeerSelectionState =
+  CardanoPeerSelectionState {
```

Comment:
I'd call this `CardanoExtraState` and then we can have & use
```hs
type CardanoPeerSelectionState = PeerSelectionState CardanoExtraState ...
```

ref: bae4f75ae3967ea2dcff75b569eb6102a0a2d928